.gitignore

```diff
@@ -1,6 +1,5 @@
 # Gradle
 build
-gradle.properties
```

Comment:
@ejona86 should this be deleted?

Reply:
I've also added `gradle.properties` with configs to enable Build Cache, parallel execution and on demand configuration (aka lazy configuration).

Reply:
That should normally be user-specific, not something that comes from the repository. So we'll want to do something different.